tests/integration/test_benchmark.py

```diff
@@ -195,3 +196,32 @@ def sample_from_synthesizer(synthesizer, n_samples):
     assert results['NewRowSynthesis'][4] == 0
     results['NewRowSynthesis'][4] = 1
     assert (results['NewRowSynthesis'] == 1).all()
+
+
+def test_benchmark_single_table_timeout():
+    """Test that benchmark times out if the ``timeout`` argument is given."""
+    # Setup
+    start_time = time.time()
+    # Run
+    scores = sdgym.benchmark_single_table(
+        synthesizers=['CTGANSynthesizer'],
+        sdv_datasets=['student_placements'],
+        timeout=5
+    )
+    total_time = time.time() - start_time
+
+    # Assert
+    assert total_time < 10.0
```

Comment:
Is this large enough? No reason to fail this test because it takes 11 seconds

Reply:
This should be good yeah, we want it to timeout as early as possible. Timeout is set to 5 and this is 2x that.